src/QirRuntime/build-qir-runtime.ps1

```diff
@@ -2,6 +2,17 @@
 # Licensed under the MIT License.
 
 if ($Env:ENABLE_QIRRUNTIME -eq "true") {
+    Write-Host "##[info]Compile Q# Projects into QIR"
+    $qirStaticPath = Join-Path $PSScriptRoot test QIR-static qsharp
```

Comment:
Beyond the scope of this PR, but we may want to consider in the future making this rely on PowerShell cmdlet functions that wrap `dotnet build` and ensure that arguments such as build configuration and assembly versions are set from environment variables in a uniform way.